MetaMask/metamask-extension
# feat: add BounceBit logo

## **Description**



To add logo to the BounceBit Network within Metamask wallet

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25049?quickstart=1)

## **Related issues**

Fixes: n/a

## **Manual testing steps**

1. Add BounceBit network to your wallet
2. You should see a logo for BounceBit displayed

## **Screenshots/Recordings**
### Before
### After

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
